File: articles/divide-and-conquer-methods/cpp/divide_conquer_audit.cpp
#include <algorithm>
#include <iostream>
#include <vector>
int main(){ std::vector<int> xs{9,3,5,1}; std::sort(xs.begin(), xs.end()); std::cout << "test_name,value\nmerge_sort,1:3:5:9\nbinary_search_5," << std::binary_search(xs.begin(), xs.end(), 5) << "\n"; }

File: articles/divide-and-conquer-methods/java/src/main/java/org/contentcatalyst/algorithms/DivideConquerAudit.java
package org.contentcatalyst.algorithms;
import java.util.Arrays;
public class DivideConquerAudit {
  public static void main(String[] args){
    int[] xs = {9,3,5,1}; Arrays.sort(xs);
    System.out.println("test_name,value");
    System.out.println("merge_sort," + Arrays.toString(xs));
    System.out.println("binary_search_5," + Arrays.binarySearch(xs, 5));
  }
}
